src/minimal-webapi-openidconnect-v2/README.md

```diff
@@ -0,0 +1,121 @@
+# Protected ASP.NET Core minimal API | Microsoft identity platform
+
+The sample code provided here has been created using minimal web API in ASP.NET Core 6.0, and slightly modified to be protected for a single organization using [ASP.NET Core Identity](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-6.0) that interacts with [Microsoft Authentication Library (MSAL)](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-overview).  In other words, a very minimalist web api is secured by adding an authorization layer before user requests can reach protected resources.  At this point it is expected that the user authentication had already happened, so a token containing user's information is being sent in the request headers to be used in the authorization process.
+
+:link: For more information about how to proctect your projects, please let's take a look at https://docs.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code. To know more about how this sample has been generated, please visit https://docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&tabs=visual-studio-code
+
+## Prerequisites
+
+1. [Download .NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
+
+## Register the web API application in your Azure Active Directory
+
+1. Choose a password for the Azure AD App
+
+   ```bash
+   AZURE_AD_APP_SECRET=<at-least-sixteen-characters-here>
```

Comment:
check whether we can create passwordless this app.

Reply:
done | addressed from [76a55](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/76a5523a155f31f5f59e33095bfe9e352ecf0744)

Reply:
But you still have the user create it in a prior step, just doesn't get used anymore.  Can we remove that bit as well?

> 1. Choose a password for the Azure AD App

Reply:
good catch @ckittel bug fixed from [9c7c2](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/2/commits/9c7c20cdc8bd5e83b918b4b2be55a38b607b836c)